db/import_pindex.go

```diff
@@ -56,7 +56,7 @@ func getListenerImportDest(ctx context.Context, indexParams string) (cbgt.Dest,
 		return nil, fmt.Errorf("error unmarshalling dbname from cbgt index params: %w", err)
 	}
 
-	base.DebugfCtx(ctx, base.KeyDCP, "Fetching listener import dest for %v", base.MD(sgIndexParams.DestKey))
+	base.DebugfCtx(ctx, base.KeyImport, "Fetching listener import dest for %v", base.MD(sgIndexParams.DestKey))
```

Comment:
This one can probably stay as `DCP` - it's CBGT-specific logging.

```suggestion
	base.DebugfCtx(ctx, base.KeyDCP, "Fetching listener import dest for %v", base.MD(sgIndexParams.DestKey))
```